config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users, :controllers => { registrations: 'registrations' }
```

Comment:
Yeah I hate hashrockets too (`=>`), this can be written as: `devise_for :users, controllers: { registrations: 'registrations' }`.